packages/scripts/config/jest-e2e.config.js

```diff
@@ -9,15 +9,16 @@ const path = require( 'path' );
 const { hasBabelConfig } = require( '../utils' );
 
 const jestE2EConfig = {
-	preset: 'jest-puppeteer',
+	globalSetup: path.join( __dirname, 'jest-environment-puppeteer', 'setup' ),
+	globalTeardown: path.join(
+		__dirname,
+		'jest-environment-puppeteer',
+		'teardown'
+	),
+	testEnvironment: path.join( __dirname, 'jest-environment-puppeteer' ),
+	setupFilesAfterEnv: [ 'expect-puppeteer' ],
 	testMatch: [ '**/specs/**/*.[jt]s', '**/?(*.)spec.[jt]s' ],
 	testPathIgnorePatterns: [ '/node_modules/' ],
-	reporters:
```

Comment:
We no longer use Travis so this code is not used. I also don't think that `travis-fold-passes-reporter.js` file is still present in the Jest preset.